pkgs/applications/graphics/pinta/default.nix

```diff
@@ -37,35 +40,39 @@ buildDotnetModule rec {
     owner = "PintaProject";
     repo = "Pinta";
     rev = version;
-    sha256 = "sha256-iOKJPB2bI/GjeDxzG7r6ew7SGIzgrJTcRXhEYzOpC9k=";
+    sha256 = "sha256-Bvzs1beq7I1+10w9pmMePqGCz2TPDp5UK5Wa9hbKERU=";
   };
 
-  postFixup = ''
-    # Rename the binary
-    mv $out/bin/Pinta $out/bin/pinta
+  # https://github.com/NixOS/nixpkgs/issues/38991
+  # bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
+  LOCALE_ARCHIVE = "${glibcLocales}/lib/locale/locale-archive";
 
-    # Copy desktop icons
-    for size in 16x16 22x22 24x24 32x32 96x96 scalable; do
```

Comment:
No need to manually copy icon again?

Reply:
Not, the icons are now installed [here](https://github.com/NixOS/nixpkgs/pull/155224/files#diff-e77b2e0a14c230e5341467fce27888c5f0481c71bce22f9dbc0f0e0957378225R71-R75).